pages/security.md

```diff
@@ -183,6 +183,64 @@ heroku config:set \
   SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT_SECRET="$SPRING_SECURITY_OAUTH2_CLIENT_REGISTRATION_OIDC_CLIENT_SECRET"
 ```
 
+#### Create a Native Okta App for Mobile
```

Comment:
I get a bit confused with this sentence.
I keep thinking that I need an additional `Native App for Mobile`.
Would it be better to change to something like `Register a Native App for Mobile on Okta`?
Or `Create a Native App for Mobile on Okta` like bellow?

Reply:
Fixed.